src/pubsub.js

```diff
@@ -50,6 +63,76 @@ class BasicPubSub extends Pubsub {
     })
   }
 
+  /**
+   * Dial a received peer.
+   * @override
+   * @param {PeerInfo} peerInfo The peer being dialed
+   * @param {function} callback
+   *
+   * @returns {void}
+   */
+  _dialPeer (peerInfo, callback) {
+    callback = callback || function noop () { }
```

Comment:
is this to prevent executing an undefined function?

Reply:
exactly!